docs/en_us/platform_api/source/courses/overview.rst

```diff
@@ -0,0 +1,64 @@
+.. _Courses API Overview:
+
+#############################
+Courses API Overview
+#############################
+
+Use the Courses API to obtain information about edX courses.
```

Comment:
I'm going to leave as is for now (in keeping with existing conventions throughout this guide) until we have a complete overhaul of API doc and decide on styles.